src/Grpc.Net.Client/GrpcClient.cs

```diff
@@ -49,6 +50,12 @@ private class Cache<TClient>
             private readonly static Func<Func<CallInvoker, TClient>> _createActivator = () =>
             {
                 var constructor = typeof(TClient).GetConstructor(new[] { typeof(CallInvoker) });
+
+                if (constructor == null)
+                {
+                    throw new InvalidOperationException($"Could not resolve a valid constructor for type {typeof(TClient)}. Please make sure it is of type Grpc.Core.ClientBase or Grpc.Core.LiteClientBase.");
```

Comment:
How about this as the error:

`$"Could not create an instance of {typeof(TClient)}. The client type must have public constructor with a {typeof(CallInvoker)} argument. A client type can be generate from a *.proto file using the Grpc.Tools package."`

Reply:
Definitely an improvement. I could mutter about other ways of creating clients, but that isn't a hill I need to die on.